src/components/Loading.tsx

```diff
@@ -0,0 +1,26 @@
+'use client';
+
+import Lottie from 'lottie-react';
+
+import loadingData from '@/../public/loading.json';
+
+interface LoadingProps {
+  width?: number;
+  height?: number;
+}
+
+const Loading = ({ width = 150, height = 100 }: LoadingProps) => {
+  return (
+    <div
+      style={{ width: `${width}px`, height: `${height}px` }}
+      className='flex justify-center items-center'
+    >
+      <Lottie
+        animationData={loadingData}
+        style={{ width: '100%', height: '100%' }}
```

Comment:
```suggestion
        className='size-full'
```

<img width="921" alt="image" src="https://github.com/user-attachments/assets/3869edb7-cc86-4480-9a06-aa9337a02c9b" />

Tailwind CSS의 `size-full`은 `Lottie` 태그에 적용이 안 되는 걸까여? Inline Style로 적용하셨길래 궁금해서 질문드립니다!

Reply:
네네 ㅠ Lottie 컴포넌트 자체에 className을 설정할 수 없도록 만들어져있는 것 같습니다..!

<img width="1103" alt="image" src="https://github.com/user-attachments/assets/63a183c9-0b9e-4df6-afd9-efa7a08f040b" />

Reply:
아항 그렇군요! 설명 감사합니다 ㅎㅎ 그럼 그대로 반영해주세요! :)